docs/accessibility_testing.md

```diff
@@ -0,0 +1,55 @@
+# Accessibility Testing in AMY
+
+## Automated Testing
+
+The automated tests aim to test one page corresponding to each template in `amy/templates` (including base templates and the `includes` subfolder).
+
+The following pages cannot be effectively tested automatically at the moment. Problems include:
+
+* page inaccessible to logged-in admin (`/terms/action_required``)
+* objects unavailable (e.g. no available signups for instructors to view)
```

Comment:
You mean that our dev_database command could create more objects? I think we would be able to implement this.

Reply:
Yes, that's what I mean. I created #2468 for this